lib/beaker/host_prebuilt_steps.rb

```diff
@@ -173,7 +173,7 @@ def sync_root_keys host, opts
         elsif host['platform'] =~ /aix/
           host.exec(Command.new(ROOT_KEYS_SYNC_CMD_AIX % "env PATH=/usr/gnu/bin:$PATH bash"), :accept_all_exit_codes => true)
         else
-          host.exec(Command.new(ROOT_KEYS_SYNC_CMD % "env PATH=/usr/gnu/bin:$PATH bash"), :accept_all_exit_codes => true)
+          host.exec(Command.new(ROOT_KEYS_SYNC_CMD % "env PATH=\"/usr/gnu/bin:$PATH\" bash"), :accept_all_exit_codes => true)
```

Comment:
I'm surprised that this works, because I thought you had to specify cygwin-ized paths, e.g. env PATH=/usr/gnu/path:/cygdrive/c/Windows/..." and cygwin translates them to Windows equivalents before calling `CreateProcess`. For example, in the AIO defaults: https://github.com/puppetlabs/beaker/blob/7439f35a4e24fc910502f32a2d3bbd9f98f07278/lib/beaker/dsl/install_utils/aio_defaults.rb#L19